tools/docker/docker-compose.paritynet.yaml

```diff
@@ -6,7 +6,7 @@ services:
       - devnet
   devnet:
     container_name: parity
-    image: smartcontract/devnet
+    image: smartcontract/devnet:st-peters
```

Comment:
Temporary change until we merge https://github.com/smartcontractkit/devnet/pull/43 and release a new latest version.